internal/scorecard/alpha/tests/basic.go

```diff
@@ -39,13 +40,17 @@ func CheckSpecTest(bundle *apimanifests.Bundle) scapiv1alpha2.ScorecardTestResul
 		return r
 	}
 
+	return checkSpecHelper(crSet, r)
```

Comment:
Nit: helper is implied
```suggestion
	return checkSpec(crSet, r)
```